http/wsserver_test.go

```diff
@@ -24,47 +24,54 @@ package http
 
 import (
 	"fmt"
+	"sync"
 	"testing"
 	"time"
 
 	"github.com/skydive-project/skydive/common"
 )
 
 type fakeServerSubscriptionHandler struct {
-	DefaultWSServerEventHandler
-	t        *testing.T
-	server   *WSServer
-	received map[string]bool
+	sync.RWMutex
+	DefaultWSClientEventHandler
+	t         *testing.T
+	server    *WSServer
+	received  int
+	connected int
 }
 
 type fakeClientSubscriptionHandler struct {
+	sync.RWMutex
 	DefaultWSClientEventHandler
-	t        *testing.T
-	received map[string]bool
+	t         *testing.T
+	received  int
+	connected int
 }
 
-func (f *fakeServerSubscriptionHandler) OnRegisterClient(c *WSClient) {
-	c.SendWSMessage(NewWSMessage("SrvValidNS", "SrvValidNSUnicast1", "AAA", "001"))
-	c.SendWSMessage(NewWSMessage("SrvNotValidNS", "SrvNotValidNSUnicast2", "AAA", "001"))
-	c.SendWSMessage(NewWSMessage("SrvValidNS", "SrvValidNSUnicast3", "AAA", "001"))
-
-	f.server.BroadcastWSMessage(NewWSMessage("SrvValidNS", "SrvValidNSBroadcast1", "AAA", "001"))
-	f.server.BroadcastWSMessage(NewWSMessage("SrvNotValidNS", "SrvNotValidNSBroacast2", "AAA", "001"))
-	f.server.BroadcastWSMessage(NewWSMessage("SrvValidNS", "SrvValidNSBroadcast3", "AAA", "001"))
+func (f *fakeServerSubscriptionHandler) OnConnected(c WSClient) {
```

Comment:
They are almost the same, true. But having wsserver_test.go makes it easier to debug simple cases